File: java-server/src/Player.java
public class Player {
    private final int clientId;
    private final int playerId;
    private int ball;

    public Player(int clientId, int playerId) {
        this.clientId = clientId;
        this.playerId = playerId;
    }

    public int getClientId() {
        return clientId;
    }

    public int getPlayerId() {
        return playerId;
    }

    public int hasBall() {
        return ball;
    }

    public void giveBall(int newBall) {
        ball = newBall;
    }
}

File: cs-server/cs-server/Player.cs
using System;

namespace CS_Server
{
    public class Player
    {
        readonly int clientId;
        readonly int playerId;
        private int _ball;

        public Player(int clientId, int playerId)
        {
            getClientId = clientId;
            getPlayerId = playerId;
        }

        public int getPlayerId { get;  }
        public int getClientId { get; }

        public int hasBall()
        {
            return _ball;
        }
		
        public void giveBall(int newBall)
        {
            _ball = newBall;
        }
    }
}